liquidctl/driver/commander_pro.py

```diff
@@ -190,12 +188,8 @@ def initialize(self, **kwargs):
             fanModes = res[1:self._fan_count+1]
             self._data.store('fan_modes', fanModes)
             status += [
-                ('Fan 1 Mode', _get_fan_mode_description(fanModes[0]), ''),
-                ('Fan 2 Mode', _get_fan_mode_description(fanModes[1]), ''),
-                ('Fan 3 Mode', _get_fan_mode_description(fanModes[2]), ''),
-                ('Fan 4 Mode', _get_fan_mode_description(fanModes[3]), ''),
-                ('Fan 5 Mode', _get_fan_mode_description(fanModes[4]), ''),
-                ('Fan 6 Mode', _get_fan_mode_description(fanModes[5]), ''),
+                ('Fan {i + 1} control mode', _fan_mode_desc(fanModes[i]), '')
```

Comment:
missing the `f` here for the format string

Reply:
Thanks, fixed.